cluster/eks.go

```diff
@@ -524,10 +589,10 @@ func (c *EKSCluster) createNodePoolsFromUpdateRequest(requestedNodePools map[str
 		if requestedNodePools[nodePool.Name] == nil {
 			updatedNodePools = append(updatedNodePools, &model.AmazonNodePoolsModel{
 				ID:        nodePool.ID,
-				CreatedBy: nodePool.CreatedBy,
```

Comment:
we need this information, don't delete it

Reply:
Not needed as we delete this nodepool from the database.